code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -176,7 +176,17 @@
 	var/datum/language/species_language = GLOB.all_languages[language]
 	return species_language.get_random_name(gender)
 
-/datum/species/proc/create_organs(mob/living/carbon/human/H) //Handles creation of mob organs.
+/datum/species/proc/create_organs(mob/living/carbon/human/H, special = FALSE) //Handles creation of mob organs.
+	var/list/special_sauce = list()
+	if(special)
+		for(var/obj/item/organ/internal/I in H.internal_organs)
+			if(I.special)
+				if(I.status == ORGAN_DEAD|ORGAN_ROBOT) // cybernetic + broken
```

Comment:
Ok since you said on discord you wanted this to only succeed if *both* are present, you need this instead:
```suggestion
				if((I.status & ORGAN_DEAD) && (I.status & ORGAN_ROBOT)) // cybernetic + broken
```

Reply:
Did that, but also inverted logic below, as we want to remove the organ if both of these are true.